src/Extsload.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 // Copyright (C) 2024 PancakeSwap
-pragma solidity ^0.8.24;
+pragma solidity ^0.8.26;
```

Comment:
do we need to bump this to compile this project with `0.8.26`? eg. what happens if external dev prefers 0.8.24 and imported our project -- we kind of force them to use 0.8.26

Reply:
You are right, my mind was a little unclear 😂